controllers/topology/desired_state.go

```diff
@@ -95,6 +96,8 @@ func computeInfrastructureCluster(_ context.Context, s *scope.Scope) (*unstructu
 		cluster:               cluster,
 		namePrefix:            fmt.Sprintf("%s-", cluster.Name),
 		currentObjectRef:      currentRef,
+		// Note: It is not possible to add an ownerRef to Cluster at this stage, otherwise the provisioning
```

Comment:
is this referring to e.g https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/26589e22fd6b57d0946a14f6535069858ef3ed9b/controllers/awscluster_controller.go#L99-L102
Do we have a doc we can point to where this is documented as a contract?